mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -74,12 +74,26 @@ def stop_app(self):
 
     def _is_app_installed(self):
         """Overrides superclass."""
+        # Check if app is installed
+        try:
```

Comment:
We seem to have this pattern in a lot of places, so can you stick this in a shared place somewhere?